src/App.vue

```diff
@@ -250,11 +266,52 @@ export default class App extends Mixins(mixins.TransactionMixin, NodeErrorMixin)
     }
   }
 
+  private applyTheme(isDark: boolean) {
```

Comment:
Let's transfer this method to utils (because it's hasn't context usage), and reuse in `AppHeaderMenu`